mcux/README

```diff
@@ -23,6 +23,7 @@ Status:
    MIMXRT1010	SDK 2.8.2 (2020-08-19)
    MIMXRT1015	SDK 2.8.2 (2020-08-19)
    MIMXRT1021	SDK 2.8.2 (2020-08-19)
+   MIMXRT1024	SDK 2.8.0 (2020-10-23)
```

Comment:
Can we update RT1024 to SDK 2.9. I think this will be the latest at the time of release.

Reply:
SDK 2.9 is version of RELEASE13 and RELEASE13 is in the first halt of next year. These codes is developed based on SDK2.8 and the last digit number 0 represents the number of times with SDK2.8.

Reply:
Can you update to the latest SDK package for 1024 available inside our internal system

Reply:
Now that the files are available in the public MCUXpresso site, can you please check we are using the latest available. I see some differences in `fsl_clock.*` files